build.sh

```diff
@@ -87,17 +95,45 @@ cd "${TARGET_REPO_NAME}"
 git checkout "${COMMIT_ID}"
 git submodule update --init
 
-# Source the dev shell to download clang-tidy and other tools.
-# Developers should *run* the dev shell, but we want to continue executing this script.
-export SHADERTRAP_SKIP_BASH=1
+case "$(uname)" in
+"Linux")
+  # Source the dev shell to download clang-tidy and other tools.
+  # Developers should *run* the dev shell, but we want to continue executing this script.
+  export SHADERTRAP_SKIP_BASH=1
 
-# Skip additional checks after building. These are part of ShaderTrap CI, and
-# are not necessary when releasing.
-export SHADERTRAP_SKIP_CHECK_COMPILE_COMMANDS=1
+  # Skip additional checks after building. These are part of ShaderTrap CI, and
+  # are not necessary when releasing.
+  export SHADERTRAP_SKIP_CHECK_COMPILE_COMMANDS=1
 
-source ./dev_shell.sh.template
+  source ./dev_shell.sh.template
 
-check_build.sh
+  check_build.sh
+  ;;
+
+"MINGW"*|"MSYS_NT"*)
+  # Needed to get EGL
+  mkdir -p "${HOME}/angle-release"
+  pushd "${HOME}/angle-release"
+    curl -fsSL -o angle-release.zip https://github.com/paulthomson/build-angle/releases/download/v-592879ad24e66c7c68c3a06d4e2227630520da36/MSVC2015-Release-x64.zip
```

Comment:
Looks like you could define (in `src/shadertrap/src/main.cc`) EGL_EGL_PROTOTYPES 0 and not define EGL_EGLEXT_PROTOTYPES, and then we could avoid needing a libEGL for compilation. Not right now, of course.